client/src/pages/Plan/RequirementsList.js

```diff
@@ -5,7 +5,7 @@ import { ProgrammeRequirementsItem } from "../../components";
 import usePlan from "./usePlan";
 
 const RequirementsList = ({ programme }) => {
-    const regulations = programme.regulations;
+    const regulations = (programme && programme.regulations) || [];
```

Comment:
I think this line is the same as 

`const regulations = programme ? programme.regulations : []`